docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-SOAPCALL.xml

```diff
@@ -6,23 +6,26 @@
       <primary>SOAPCALL</primary>
     </indexterm></title>
 
-  <para><emphasis>result</emphasis><emphasis role="bold"> :=
-  SOAPCALL<indexterm>
-      <primary>SOAPCALL Function</primary>
-    </indexterm>( [ </emphasis><emphasis>recset, </emphasis><emphasis
-  role="bold">]</emphasis><emphasis> url, service, instructure,
-  </emphasis><emphasis role="bold">[ </emphasis><emphasis>transform,
-  </emphasis><emphasis role="bold">]</emphasis><emphasis role="bold">
-  DATASET<indexterm>
-      <primary>DATASET</primary>
-    </indexterm>(</emphasis><emphasis>outstructure</emphasis><emphasis
-  role="bold">) </emphasis><emphasis>| outstructure </emphasis><emphasis
-  role="bold">[</emphasis><emphasis>, options </emphasis><emphasis
-  role="bold">]</emphasis> <emphasis role="bold">[, UNORDERED |
+  <para><emphasis>result</emphasis><emphasis role="bold"> </emphasis><emphasis
```

Comment:
Is there any way of preventing these extra changes (adding redundant emphasis tags)?  It makes it harder to spot the real changes.

Reply:
there is an unused set of role=bold tags on line 9 - could be removed for legibility.